node/src/Cardano/Node/API.hs

```diff
@@ -340,7 +295,7 @@ instance DB.MonadDBRead (ReaderT InfoCtx IO) where
     dbGetSerUndo  = DB.dbGetSerUndoRealDefault
     dbGetSerBlund = DB.dbGetSerBlundRealDefault
 
-
+-- can throw DBError (DBMalformed)
```

Comment:
So, the only error that the node stuff can throw is if the whole database is borked. Which is a 500 error, really, and probably not something the API needs to represent specially.

Reply:
This and a `404` when there's no "new update available" I believe.